cmd/admin-revoker/main.go

```diff
@@ -305,6 +535,31 @@ func main() {
 			fmt.Printf("%d: %s\n", k, revocation.ReasonToString[k])
 		}
 
+	case (command == "private-key-block" || command == "private-key-revoke") && len(args) == 1:
+		// 1: keyPath
+		keyPath := args[0]
+
+		_, publicKey, err := privatekey.Load(keyPath)
+		cmd.FailOnError(err, "Failed to load the provided private key")
+		r.log.AuditInfo("The provided private key has been successfully verified")
+
+		spkiHash, err := getPublicKeySPKIHash(publicKey)
+		cmd.FailOnError(err, "While obtaining the SPKI hash for the provided key")
+
+		count, err := r.countCertsMatchingSPKIHash(spkiHash)
+		cmd.FailOnError(err, "While retrieving a count of certificates matching the provided key")
+		r.log.AuditInfof("Found %d certificates matching the provided key", count)
+
+		if command == "private-key-block" {
+			err := privateKeyBlock(r, *dryRun, count, spkiHash, keyPath)
+			cmd.Fail(err.Error())
```

Comment:
I'd use `cmd.FailOnError` here, to be consistent with other usages higher up in this file (e.g. line 492).

Reply:
Every error case from the privateKeyBlock and privateKeyRevoke has a user friendly error message and embeds the precipitating error as well.

Reply:
Yeah, but from an ergonomics standpoint this requires the reader to have the extra realization "oh wait, `cmd.Fail()` isn't going to actually fail if its argument is `nil`, whereas `cmd.FailOnError()` makes that explicit.

Reply:
Ah, I see your point.